compiler/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -1406,4 +1421,133 @@ mod tests {
             }
         }
     }
+
+    fn assert_fv_attribute_lexes_to_tokens(attribute_text: &str, expected_tokens: Vec<Token>) {
+        let mut lexer = Lexer::new(attribute_text);
+
+        for token in expected_tokens.into_iter() {
+            let got = lexer.next_token().unwrap();
+            assert_eq!(got, token);
+        }
+    }
+
+    #[test]
+    fn ensures_attribute_lexes() {
+        let input = "#[ensures(result == 4)]\n";
+        let expected = vec![
+            Token::Ensures,
+            Token::LeftParen,
+            Token::Ident("result".to_string()),
+            Token::Equal,
+            Token::Int(4_i128.into()),
+            Token::RightParen,
+            Token::RightBracket,
+            Token::EOF,
+        ];
+
+        assert_fv_attribute_lexes_to_tokens(input, expected);
+    }
+
+    #[test]
+    fn multiple_attributes() {
```

Comment:
nit: multiple_attributes_lex or multiple_attributes_are_accepted

alternatively, drop "lexes" from the other names

Reply:
Dropped "lexes" from other names.